Storage/test.cc

```diff
@@ -184,6 +186,40 @@ TEST(TransactionalStorage, SmokeTest) {
       }).Go();
       EXPECT_TRUE(WasCommitted(result));
     }
+    {
```

Comment:
Let's add comment here and in the blocks above in this test, what we are actually testing in each case.